Repo: main branch build breaks locally

### Suggestion

This started happening to my `yarn` / `yarn build`s on multiple dev machines after the 4.9 upgrade:

```ts
    ✖  nx run @typescript-eslint/visitor-keys:build
$ tsc -b tsconfig.build.json
       src/visitor-keys.ts(153,3): error TS2322: Type '{ AccessorProperty: readonly ["decorators", "key", "typeAnnotation", "value"]; ArrayPattern: ("decorators" | "typeAnnotation" | "elements")[]; ArrowFunctionExpression: readonly ["typeParameters", "params", "returnType", "body"]; ... 101 more ...; TSVoidKeyword: never[]; }' is not assignable to type 'AdditionalKeys'.
         Object literal may only specify known properties, and 'AccessorProperty' does not exist in type 'AdditionalKeys'.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

...merge conflict from something? Or maybe an issue with Nx caching?

--- a/packages/ast-spec/project.json
+++ b/packages/ast-spec/project.json
@@ -3,6 +3,9 @@
   "type": "library",
   "implicitDependencies": [],
   "targets": {
+    "build": {
+      "outputs": ["packages/ast-spec/dist/*.ts"]
+    },
     "lint": {
       "executor": "@nrwl/linter:eslint",
       "outputs": ["{options.outputFile}"],

--- a/packages/ast-spec/project.json
+++ b/packages/ast-spec/project.json
@@ -9,7 +9,7 @@
         "cwd": "packages/ast-spec",
         "commands": ["yarn build"]
       },
-      "outputs": ["packages/ast-spec/dist/*.ts"]
+      "outputs": ["packages/ast-spec/dist/**/*.ts"]
     },
     "lint": {
       "executor": "@nrwl/linter:eslint",

--- a/packages/ast-spec/project.json
+++ b/packages/ast-spec/project.json
@@ -4,6 +4,9 @@
   "implicitDependencies": [],
   "targets": {
     "build": {
+      "cwd": "packages/ast-spec",
+      "commands": ["yarn build"],
+      "executor": "nx:run-commands",
       "outputs": ["packages/ast-spec/dist/*.ts"]
     },
     "lint": {

--- a/packages/ast-spec/project.json
+++ b/packages/ast-spec/project.json
@@ -4,9 +4,11 @@
   "implicitDependencies": [],
   "targets": {
     "build": {
-      "cwd": "packages/ast-spec",
-      "commands": ["yarn build"],
       "executor": "nx:run-commands",
+      "options": {
+        "cwd": "packages/ast-spec",
+        "commands": ["yarn build"]
+      },
       "outputs": ["packages/ast-spec/dist/*.ts"]
     },
     "lint": {

